CONTRIBUTING.md

```diff
@@ -124,4 +124,10 @@ Examples:
 
 #### Releases
 
-Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md).
+Please refer to the documentation within this repo's [RELEASE.md](https://github.com/jupyterhub/ltiauthenticator/blob/main/RELEASE.md). The example below updates the release to `1.3.0` and outputs the results (`--verbose`):
+
+```bash
+bump2version --verbose --new-version 1.3.0 -
+```
+
+Use the `--dry-run` flag to emulate the update without actually changing the files.
```

Comment:
What do you think about removing this besides the reference to RELEASE.md, and letting that document contain everything needed?

I'm worried that this example isn't helpful by itself, and/or the --dry-run flag idea is not part of RELEASE.md.

Reply:
Yeah, that makes sense, I'll remove it.